test/parallel/test-cluster-net-listen-ipv6only-false.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasIPv6)
+  common.skip('no IPv6 support');
+
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+const Countdown = require('../common/countdown');
+
+// This test ensures that the dual-stack support still works for cluster module
+// when `ipv6Only` is not `true`.
+const host = '::';
+const WORKER_ACCOUNT = 3;
+
+if (cluster.isMaster) {
+  const workers = new Map();
+  let address;
+
+  const countdown = new Countdown(WORKER_ACCOUNT, common.mustCall(() => {
```

Comment:
It's unnecessary to wrap the `Countdown` callback in a `mustCall()`. That's done automatically.